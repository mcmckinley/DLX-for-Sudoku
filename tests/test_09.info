Hard 16x16 A