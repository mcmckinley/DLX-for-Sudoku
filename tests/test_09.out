7  2  14 12  3  11 16 4   8  15 9  1   6  10 5  13 
3  16 9  15  5  10 12 1   7  2  6  13  8  4  11 14 
4  1  11 8   13 6  7  9   5  14 10 3   15 12 2  16 
6  5  10 13  8  2  15 14  16 12 11 4   9  1  3  7  

9  13 5  4   1  3  10 2   14 11 16 8   12 6  7  15 
15 12 16 6   11 14 9  13  4  3  7  5   10 2  8  1  
2  8  1  7   16 12 5  6   13 10 15 9   14 11 4  3  
10 14 3  11  7  4  8  15  2  1  12 6   5  16 13 9  

14 11 8  1   15 5  2  12  10 7  4  16  3  13 9  6  
5  7  13 2   14 16 6  10  3  9  8  11  1  15 12 4  
16 9  15 3   4  13 1  11  12 6  5  14  2  7  10 8  
12 4  6  10  9  8  3  7   15 13 1  2   16 5  14 11 

11 3  12 16  10 15 14 8   1  5  13 7   4  9  6  2  
8  15 7  9   2  1  11 5   6  4  14 10  13 3  16 12 
1  10 4  14  6  9  13 3   11 16 2  12  7  8  15 5  
13 6  2  5   12 7  4  16  9  8  3  15  11 14 1  10 
